### DISCUSSION TITLE
Error : Please upgrade your Copilot extension to continue using this service

### DISCUSSION THREAD

**Discussion by liuliuball** [community/community#18322] [Copilot Conversations] [answered]
VS code：

I installed the extension as instructed, but got the error below, and there's no version to be updated.

[INFO] [default] [2022-06-11T02:42:25.213Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-11T02:42:25.388Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 174 ms
[INFO] [fetch] [2022-06-11T02:42:25.393Z] Please upgrade your Copilot extension to continue using this service.

**Comment by MrKrishnaAgarwal** [comment]
https://github.com/features/copilot/signup
Please try signing up here and then download Co-pilot using it.
I hope it helps.

Download Co-pilot in VS code- https://github.com/github/copilot-docs/blob/main/docs/visualstudiocode/gettingstarted.md#getting-started-with-github-copilot-in-visual-studio-code

**Comment by davecheney** [comment, marked as answer]
@liuliuball I see you were running version 1.17.5850 and have upgraded and Copilot is now providing completions to you.
